### PR TITLE
trash page

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
This PR includes 

- trash page
- User can permanently delete the note 
- User can restore deleted notes

### CODE REVIEW

**Review by rehanmohiuddin — APPROVED** [review]
Liked The Naming Conventions Followed

File: src/pages/Label/Label.tsx [hunk 1]
```diff
@@ -24,7 +25,24 @@ export const Label = () => {
...
+              <div>
+                <h2 className="text-xl font-semibold text-xl mb-3">
+                  {note.title}
+                </h2>
+                <p>{note.noteText}</p>
+              </div>
+
+              <div className="my-2 flex">
+                {note.labels.map((label) => (
+                  <LabelTag currentLabel={label} noteId={note.id} />
```
Comment (rahulyadav139) [inline src/pages/Label/Label.tsx:41]: suggestion: it seems you forgot to write "key" attribute.

**Review by tanishk15sharma — APPROVED** [review]
clean code 
naming convention is good 
modular and small components

File: src/components/TrashCard/TrashCard.tsx [hunk 2]
```diff
@@ -2,13 +2,61 @@ import { MdOutlineDelete, MdOutlineRestoreFromTrash } from "react-icons/md";
...
+      const updatedNoteList = notes.filter((note) => note.id !== id);
+
+      await addNote(updatedNoteList, uid);
+      setNotes(updatedNoteList);
+      toast.success("Note deleted permanently");
+      setIsInProcess(false);
+    } catch (error) {
+      console.log(error);
+      toast.error("Please try again later.");
+      setIsInProcess(false);
```
Comment (rahulyadav139) [inline src/components/TrashCard/TrashCard.tsx:28]: suggestion:  you can write "setIsInProcess" in finally block.

**Review by rahulyadav139 — APPROVED** [review]
Appreciation: Everything is looking great. codes are readable. great work.

**Review by abhij1607 — COMMENTED** [review]
Good folder structure
Neat & clean code

Can be merged

File: src/components/TrashCard/TrashCard.tsx (hunk 2 above)
Comment (junaidshaikh-js) [reply src/components/TrashCard/TrashCard.tsx:28]: Noted, I will change in later PR

File: src/pages/Label/Label.tsx (hunk 1 above)
Comment (junaidshaikh-js) [reply src/pages/Label/Label.tsx:41]: Noted, changes will be made in future PRs

**Pull request merged**